examples/scripts/project/categorization/bootstrap_taxonomy.py

```diff
@@ -0,0 +1,169 @@
+"""Example script for bootstrapping a Tamr Categorization project with taxonomy categories"""
+import argparse
+import logging
+import pandas as pd
+import requests
+
+from typing import Dict, Any
+
+import tamr_toolbox as tbox
+from tamr_toolbox.project.schema_mapping import schema
+
+# Uncomment for bootstrapping categories at a specific taxonomy tier
+# from tamr_toolbox.project.categorization import metrics
+
+
+LOGGER = logging.getLogger(__name__)
+
+CATEGORY_ATTRIBUTE_NAME = "Category Name"
+FULL_PATH_LIST_NAME = "Full Path List"
+FULL_PATH_STR_NAME = "Full Path"
+COLUMN_NAMES = [FULL_PATH_STR_NAME, FULL_PATH_LIST_NAME]
+PK_NAME = "Primary Key"
+
+
+def main(*, instance_connection_info: Dict[str, Any], categorization_project_id: str) -> bool:
+    """
+    Bootstraps taxonomy in a categorization project
+
+    Args:
+        instance_connection_info: Information for connecting to Tamr (host, port, username etc)
+        categorization_project_id: The id of the target categorization project
+
+    Returns: Boolean indicating whether boostrap was successful or not
+
+    Raises:
+        TypeError: retrieved project is not a categorization project
+
+    """
+
+    # Create the tamr client
+    tamr_client = tbox.utils.client.create(**instance_connection_info)
+
+    # Retrieve the project
+    project = tamr_client.projects.by_resource_id(categorization_project_id)
+    LOGGER.info(f"Retrieved project with name: {project.name}")
+
+    # Get the project taxonomy
+    # NOTE: .taxonomy() function breaks on newer tamr instances SUP-5075
+    try:
+        project.taxonomy()
+    except requests.exceptions.RequestException:
+        no_taxonomy_error = f"Project {project.name} is not associated with any taxonomy yet."
+        raise RuntimeError(no_taxonomy_error)
+    LOGGER.info(f"Retrieved project taxonomy with name: {project.taxonomy().name}")
+
+    # Bootstrap all available categories
+    categories = project.taxonomy().categories()
+    category_list = []
+    for category in categories:
+        category_list.append(category.path)
+    # For bootstrapping at a specific tier = -1 (leaf), 1, 2, ... use the existing function
+    # category_set = metrics._get_categories_at_tier(project=project, tier=1)
+    # category_list = [category.split("|") for category in category_set]
```

Comment:
Remove and possibly create a separate example

Reply:
Do you mean a separate example in the same script, since the rest of the steps are the same?

Reply:
Was suggesting a separate example, but don't have to if you think would be not useful